### PR TITLE
Documenting sql store license methods

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Documenting sql store license methods

### CODE REVIEW

File: store/sqlstore/license_store.go [hunk 1]
```diff
@@ -29,6 +31,12 @@ func NewSqlLicenseStore(sqlStore SqlStore) store.LicenseStore {
 func (ls SqlLicenseStore) CreateIndexesIfNotExists() {
 }
 
+// Save validates and stores the license instance in the database. The Id
```
Comment (justinegeffen) [inline store/sqlstore/license_store.go]:
```suggestion
// Saves, validates, and stores the license instance in the database. The Id
```

File: store/sqlstore/license_store.go [hunk 2]
```diff
@@ -29,6 +31,12 @@ func NewSqlLicenseStore(sqlStore SqlStore) store.LicenseStore {
...
+// and Bytes fields are mandatory. The Bytes field is limited to a maximun
```
Comment (justinegeffen) [inline store/sqlstore/license_store.go]:
```suggestion
// and Bytes fields are mandatory. The Bytes field is limited to a maximum
```

File: store/sqlstore/license_store.go [hunk 3]
```diff
@@ -29,6 +31,12 @@ func NewSqlLicenseStore(sqlStore SqlStore) store.LicenseStore {
...
+// of 10000 bytes. If the license Id match with an existing license in the
```
Comment (justinegeffen) [inline store/sqlstore/license_store.go]:
```suggestion
// of 10000 bytes. If the license Id matches with an existing license in the
```

File: store/sqlstore/license_store.go [hunk 4]
```diff
@@ -29,6 +31,12 @@ func NewSqlLicenseStore(sqlStore SqlStore) store.LicenseStore {
...
+// database it returns the license stored in the database. If not, it save the
```
Comment (justinegeffen) [inline store/sqlstore/license_store.go]:
```suggestion
// database it returns the license stored in the database. If not, it saves the
```

File: store/sqlstore/license_store.go [hunk 5]
```diff
@@ -46,6 +54,9 @@ func (ls SqlLicenseStore) Save(license *model.LicenseRecord) (*model.LicenseReco
 	return &storedLicense, nil
 }
 
+// Get obtains from the database the license with the provided id parameter.
```
Comment (justinegeffen) [inline store/sqlstore/license_store.go]:
```suggestion
// Obtains the license with the provided ID parameter from the database. 
```

File: store/sqlstore/license_store.go (hunk 1 above)
Comment (jespino) [reply store/sqlstore/license_store.go]: Actually what I'm saying here is `Save` as function, "validates and stores the license instance"

File: store/sqlstore/license_store.go (hunk 5 above)
Comment (jespino) [reply store/sqlstore/license_store.go]: The functions documentation (by design in golang documentation system) must start with the function name, so in this case must start with the `Get` and then the explanation of what `Get` does.

Comment (jespino) [reply store/sqlstore/license_store.go]: The parameter itself is `id` with lowercase, we are referring the parameter name here, so it should match with the casing too.

File: store/sqlstore/license_store.go (hunk 1 above)
Comment (justinegeffen) [reply store/sqlstore/license_store.go]: Ah, got it. You can disregard this change then. :)

File: store/sqlstore/license_store.go [hunk 6]
```diff
@@ -46,6 +54,9 @@ func (ls SqlLicenseStore) Save(license *model.LicenseRecord) (*model.LicenseReco
 	return &storedLicense, nil
 }
 
+// Get obtains the license with the provided id parameter from the database.
```
Comment (justinegeffen) [inline store/sqlstore/license_store.go]:
```suggestion
// Get obtains the license with the provided ID parameter from the database.
```

File: store/sqlstore/license_store.go [hunk 7]
```diff
@@ -46,6 +54,9 @@ func (ls SqlLicenseStore) Save(license *model.LicenseRecord) (*model.LicenseReco
...
+// If the license doesn't exists it returns a model.AppError with
```
Comment (justinegeffen) [inline store/sqlstore/license_store.go]:
```suggestion
// If the license doesn't exist it returns a model.AppError with
```

File: store/sqlstore/license_store.go (hunk 6 above)
Comment (jespino) [reply store/sqlstore/license_store.go]: The parameter name here is `id` not `ID`, is more accurate to say `id` parameter because

**Pull request merged**